resources/zib/zib-TreatmentDirective2-ServiceRequest.xml

```diff
@@ -0,0 +1,149 @@
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TreatmentDirective2.ReasonForEnding" />
+      </type>
+      <mapping>
+        <identity value="zib-treatmentdirective2-v1.0-2020EN" />
+        <map value="NL-CM:2.2.7" />
+        <comment value="ReasonForEnding" />
+      </mapping>
+    </element>
+    <element id="ServiceRequest.modifierExtension:specificationOther">
+      <path value="ServiceRequest.modifierExtension" />
+      <sliceName value="specificationOther" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TreatmentDirective2.SpecificationOther" />
+      </type>
+      <condition value="zib-TreatmentDirective2-2" />
+      <mapping>
+        <identity value="zib-treatmentdirective2-v1.0-2020EN" />
+        <map value="NL-CM:2.2.4" />
+        <comment value="specificationOther" />
+      </mapping>
+    </element>
+    <element id="ServiceRequest.modifierExtension:specificationOther.value[x]">
+      <path value="ServiceRequest.modifierExtension.value[x]" />
+      <condition value="zib-TreatmentDirective2-2" />
+    </element>
+    <element id="ServiceRequest.status">
+      <path value="ServiceRequest.status" />
+      <comment value="This element is implicitly mapped to the zib concepts NL-CM:2.2.5 (MostRecentReviewDate) and NL-CM:2.2.6 (DateExpired). Unless the status is explicitly recorded, the following guidance applies:&amp;#xD;&amp;#xA;&amp;#xD;&amp;#xA;* When MostRecentReviewDate is in the past and DateExpired is absent or in the future, `.status` will usually be set to _active_.&amp;#xD;&amp;#xA;* When DateExpired is in the past, `.status` will usually be set to _revoked_.&amp;#xD;&amp;#xA;* When MostRecentReviewDate is absent or in the future,  which is not possible following the zib, `.status` will usually be set to _draft_." />
+      <mapping>
+        <identity value="zib-treatmentdirective2-v1.0-2020EN" />
+        <map value="NL-CM:2.2.5" />
+        <comment value="MostRecentReviewDate (implicit, main mapping is on `Consent.dateTime`)" />
+      </mapping>
+    </element>
+    <element id="ServiceRequest.intent">
+      <path value="ServiceRequest.intent" />
+      <definition value="The resource will be used for TreatmentDirective purposes so the fixed value will always be directive." />
+      <fixedCode value="directive" />
+    </element>
+    <element id="ServiceRequest.category.coding.system">
+      <path value="ServiceRequest.category.coding.system" />
+      <fixedUri value="http://snomed.info/sct" />
+    </element>
+    <element id="ServiceRequest.category.coding.code">
+      <path value="ServiceRequest.category.coding.code" />
+      <fixedCode value="11291000146105" />
+    </element>
+    <element id="ServiceRequest.doNotPerform">
+      <path value="ServiceRequest.doNotPerform" />
+      <meaningWhenMissing value="If missing, the specificationOther from the modifierExtension SHALL be used." />
```

Comment:
This is forbidden to use in a profile (and for good reason; you cannot change the meaning of a resource using a profile). See: https://www.hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.meaningWhenMissing. I'll remove it.